documentation/release_notes.markdown

```diff
@@ -1,147 +1,157 @@
 ---
 layout: default
-title: "Puppet Server: Release Notes"
+title: "Puppet Server 2.0: Release Notes"
 canonical: "/puppetserver/latest/release_notes.html"
 ---
 
-## Puppet Server 1.0.2
+## Puppet Server 2.0
 
-The 1.0.2 release of Puppet Server includes several bug fixes. It also improves logging functionality by allowing Logback changes to take effect without a restart.
+In keeping with [semantic versioning][semver] practices, this release
+of Puppet Server introduces changes that break compatibility with
+Puppet 3.x agents. Please carefully read these release notes before
+attempting to upgrade your Puppet installation, as the locations of
+key files have moved, and changes to the Puppet Server API may require
+configuration changes.
 
-### Bug Fixes
+### Supported Platforms
+
+Puppet Server 2.0 ships for the following supported platforms:
 
-#### Filebucket files treated as binary data
+ * Enterprise Linux 7
+ * Enterprise Linux 6
+ * Ubuntu 14.04
+ * Ubuntu 12.04
+ * Debian 7
 
-Puppet Server now treats filebucket files as binary data. This prevents possible data alteration resulting from Puppet Server inappropriately treating all filebucket files as text data.
+### What's New in Puppet Server 2.0 
 
-* [SERVER-269](https://tickets.puppetlabs.com/browse/SERVER-269): Puppet Server aggressively coerces request data to UTF-8
+#### Package Name
 
-#### `puppetserver gem env` command now works
+Puppet Server's package name is no longer `puppet-server`, it is
```

Comment:
I think it was already "puppetserver" for all platforms.  We changed the name of the Maven artifact from "puppet-server" to "puppetserver" but I believe the package that users installed from public repos for both apt and yum was "puppetserver" for Puppet Server 1.x.  @jeffmccune - can you confirm?